man/systemd-detect-virt.xml

```diff
@@ -155,6 +155,11 @@
             <entry><varname>rkt</varname></entry>
             <entry>rkt app container runtime</entry>
           </row>
+
+          <row>
+            <entry><varname>acrn</varname></entry>
+            <entry>ACRN hypervisor</entry>
```

Comment:
Hmm, could you add `<ulink url="https://projectacrn.org">` here? I think ACRN is not that well know.

(If you could add links to some of other ones, that'd be even nicer.)